hooks/yalc/yalc-check.sh

```diff
@@ -2,4 +2,8 @@
 # Exits with status code 1 if a yalc dependency is present in the package manifest
 set -e
 
-yalc check
+for file in "$@" ; do
+    if egrep "(file|link):\.yalc" $file; then
+        exit 1
```

Comment:
I think that it would be good to `echo` a useful message to standard error.  It makes it easier for people who see the issue to know what to do.